src/vs/workbench/contrib/chat/browser/chatContentParts/toolInvocationParts/chatTerminalToolProgressPart.ts

```diff
@@ -688,40 +651,38 @@ class ChatTerminalToolOutputSection extends Disposable {
 		return this._container.classList.contains('expanded');
 	}
 
-	private readonly _container: HTMLElement;
-	private readonly _title: HTMLElement;
-	private readonly _displayCommand: string;
-	private readonly _terminalData: IChatTerminalToolInvocationData;
-	private readonly _accessibleViewService: IAccessibleViewService;
 	private readonly _onDidChangeHeight: () => void;
 	private readonly _ensureTerminalInstance: () => Promise<ITerminalInstance | undefined>;
 	private readonly _resolveCommand: () => ITerminalCommand | undefined;
-	private readonly _getTerminalTheme: () => { background?: string; foreground?: string } | undefined;
-	private readonly _getStoredCommandId: () => string | undefined;
 
 	private readonly _outputBody: HTMLElement;
 	private _outputScrollbar: DomScrollableElement | undefined;
-	private _outputContent: HTMLElement | undefined;
 	private _outputResizeObserver: ResizeObserver | undefined;
```

Comment:
Don't think we need either of these now? Scrollbar should be handled by xterm and we know the times when xterm's size changes as it's either cols/rows change, zoom or font config?

Reply:
we do still need the scrollable container, especially for the horizontal scrolling. it's badly named, will fix.